man/fancy_sv_sizedens.Rd

```diff
@@ -19,8 +19,8 @@ fancy_sv_sizedens(
   hide_legend = FALSE,
   chr_select = paste0("chr", c(1:22)),
   plot_title = paste0(this_sample),
- 
-    plot_subtitle = paste0("SV sizes for Manta calls. Dashed line annotates mean variant size.\\nVAF cut off: ",
+  plot_subtitle =
```

Comment:
Did you use some kind of auto-formatting for this? Was not aware that this is the preferred syntax format.